src/backend/expungeservice/expunger.py

```diff
@@ -30,15 +30,20 @@ def run(record: Record) -> Dict[str, TimeEligibility]:
         cases = analyzable_record.cases
         for charge in analyzable_record.charges:
             eligibility_dates: List[Tuple[date, str]] = []
-            other_blocking_charges = [
-                c
-                for c in analyzable_record.charges
-                if c.charge_type.blocks_other_charges and c.id != charge.id and c.edit_status != EditStatus.DELETE
-            ]
-            dismissals, convictions = Expunger._categorize_charges(other_blocking_charges)
-            most_recent_blocking_dismissal = Expunger._most_recent_different_case_dismissal(charge, dismissals)
-            most_recent_blocking_conviction = Expunger._most_recent_convictions(convictions)
-            other_convictions_all_traffic = Expunger._is_other_convictions_all_traffic(convictions)
+
+            other_charges = [c for c in analyzable_record.charges
+                             if c.id != charge.id and c.edit_status != EditStatus.DELETE]
+
+            other_blocking_charges = [c for c in other_charges
+                                      if c.charge_type.blocks_other_charges]
+
+            dismisals_all, convictions_all = Expunger._categorize_charges(other_charges)
```

Comment:
```suggestion
            _, convictions_all = Expunger._categorize_charges(other_charges)
```
Since this variable isn't used.